tini/src/seccomp_fd_notify.c

```diff
@@ -123,9 +130,27 @@ static int install_notify_filter(void)
 		BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_ALLOW),
 	};
 
-	struct sock_fprog prog = {
-		.len = (unsigned short)(sizeof(filter) / sizeof(filter[0])),
-		.filter = filter,
+struct sock_filter net_perf_filter[] = {
```

Comment:
This variable is called `net_perf`, but it only has `net`.

I think we'll need ti to *also* have perf, unless you want to concat them somehow?

Reply:
Also can you add sendmmesg too?